MahApps.Metro/Controls/RangeSlider.cs

```diff
@@ -2025,17 +2025,15 @@ private Double CalculateNextTick(Direction dir, double chekingValue, double dist
             if (!IsMoveToPointEnabled)
             {
                 //Check if current value is exactly Tick value or it situated between Ticks
-                if (!IsDoubleCloseToInt((chekingValue - Minimum) / TickFrequency))
+                if (!IsDoubleCloseToInt((chekingValue + distance - Minimum) / TickFrequency))
                 {
-                    double x = (chekingValue - Minimum) / TickFrequency;
+                    double x = (chekingValue + distance - Minimum) / TickFrequency;
```

Comment:
Following on from the earlier comment, we're re-calculating this value a second time. Could we simplify it further to reuse the last value?